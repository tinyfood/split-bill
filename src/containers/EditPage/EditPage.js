import React from 'react'
import FriendsInput from '../../components/FriendsInput/FriendsInput'
import TipsInput from '../../components/TipsInput/TipsInput'
import BillInput from '../../components/BilInput/BillInput'
import styled from 'styled-components'

const EditPageWrap = styled.div`
  width: 96%;
  margin: 0 auto;
`

const EditPage = () => {
  return (
    <EditPageWrap>
      <FriendsInput />
      <TipsInput />
      <BillInput />
    </EditPageWrap>
  )
}

export default EditPage
