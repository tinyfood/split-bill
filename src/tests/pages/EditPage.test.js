import React from 'react'
import { shallow } from 'enzyme'
import toJSON from 'enzyme-to-json'
import EditPage from '../../pages/EditPage/EditPage'

test('should render EditPage correctly', () => {
  const wrapper = shallow(<EditPage />)
  expect(toJSON(wrapper)).toMatchSnapshot()
})
