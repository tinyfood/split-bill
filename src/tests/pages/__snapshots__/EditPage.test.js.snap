// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render EditPage correctly 1`] = `
<styled.div>
  <ConnectFunction />
  <ConnectFunction />
  <ConnectFunction />
</styled.div>
`;
